charts/telegraf/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: telegraf
-version: 1.8.28
+version: 1.8.29
 appVersion: 1.26.3
```

Comment:
Might as well use latest version of telegraf?
```suggestion
appVersion: 1.27.3
```

Reply:
Looks like somebody else beat me to the version update.